### DISCUSSION TITLE
king-dahmanus's training errors

### DISCUSSION THREAD

**Discussion by king-dahmanus** [RHVoice/RHVoice#590] [Q&A]
this is a thread for all my issues, will begin in the second comment

**Comment by zstanecic** [comment]
Кувщцтдщфв еру ызелю

 

From: king-dahmanus ***@***.***> 
Sent: Friday, July 1, 2022 11:29 AM
To: RHVoice/RHVoice ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [RHVoice/RHVoice] error when trying to extract SPTK-3.11.tar.gz (Issue #589)

 

I'm trying to prepare sptk, but I followed the tutorial correctly and curled the file. But then when I tried extracting it with tar -xvf SPTK-3.11.tar.gz, I got
tar: This does not look like a tar archive
gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
why is it doing this? I'm sure it's in the correct gzip format, so what's wrong here?

**Comment by king-dahmanus** [comment]
actually please don't close it, because I ran into another error while making htk
HGraf.c:73:10: fatal error: X11/Xlib.h: No such file or directory                                                                                                       
by the way, this will be the thread for all my errors

**Comment by zstanecic** [comment]
Please install

Sudo apt install x11-dev 

And it will be fixed.

**Comment by king-dahmanus** [comment]
it's not working, unable to find package x11-dev

**Comment by zstanecic** [comment]
Sorry, it's called libx11-dev

**Comment by king-dahmanus** [comment]
thanks. I should've known, with all the necessary packages starting with lib

**Comment by king-dahmanus** [comment]
I've got another error, with the hts slt demo step. So, up to that point, I've changed all paths from tts/* to tts/rhvoice/* as that's the correct path where I set rhvoice stuff. The error is:
configure: error: Can't find text2utt of festival                                                                                                                       
the command is:
help@DESKTOP-3QACTUK:~/tts/rhvoice/HTS-demo_CMU-ARCTIC-SLT$ ./configure --with-fest-search-path=$(echo ~)/tts/rhvoice/festival/examples   --with-sptk-search-path=$(echo~)/tts/rhvoice/SPTK-3.11/build/bin   --with-hts-engine-search-path=$(echo ~)/tts/rhvoice/hts_engine_api110/bin   --with-hts-search-path=$(echo ~)/tts/rhvoice/hts23/bin
Help please?

**Comment by king-dahmanus** [comment]
yeah this is good. Awaiting replies to my error

**Comment by king-dahmanus** [comment]
I don't know what github did to the command when I pasted it. Strange. I'm sure I had equals and left parens and backslashes before. Did the fire of github evaporate them?

**Comment by king-dahmanus** [comment]
it seams there is text2wave.sh in the examples directory of festival, but it's not recognizing it for some reason

**Comment by king-dahmanus** [comment]
bump. Please, some one, can you help me?

**Comment by zstanecic** [comment]
You don't need to do the hts demo step. Just do the steps described in part 3. I am not bothering with p2 anymore myself, as it is optional.

**Comment by king-dahmanus** [comment]
ah, so even if I changed the paths, as long as they were always correct it will work?

**Comment by zstanecic** [comment]
Don’t change the paths if not instructed to do so.

If your environment was in /home/dahmanus/tts/

Let it be there for ever.

**Comment by king-dahmanus** [comment]
too late, I changed it in all the steps because /home/tts has festvox stuff because rhvoice isn't the only thing I plan to work with, and it certainly isn't as automated as festvox training, maybe if @Olga-Yakovleva could make a script that automates all steps after guessing the f0 range that would make it easier for everyone, what do you think.
oh, another error, now with the paths again. -bash: syntax error near unexpected token `ame]=self.settings["bindir"]'                                                                                                
command:
		for name in ["--with-sptk-search-path","--with-hts-search-path","--with-hts-engine-search-path"]:
			params[name]=self.settings["bindir"]

**Comment by king-dahmanus** [comment]
please help you guys, or I'm forced to remove all the folders and only keep the zips and do it from scratch, on another folder perhaps? But not tts. Maybe a new user entirely

**Comment by king-dahmanus** [comment]
ok so now I'm stuck. I reinstalled the environment and didn't change the paths this time, but I ran to the same problem as before
the command:
		for name in ["--with-sptk-search-path","--with-hts-search-path","--with-hts-engine-search-path"]:
			params[name]=self.settings["bindir"]
the result:
a weird promt with a greater than > sign which doesn't respond to any command except control c after a while. Am I supposed to run ./configure normally? Or what am I supposed to do in this step

**Comment by king-dahmanus** [comment]
even when I do run ./configure, it says:
configure: error: Can't find text2utt of festival

**Comment by king-dahmanus** [comment]
hello? I believe I have to do the whole process from the beginning, or maybe give up on it untill all voices get retrained to use world and the training system adapts to that procedure?
